README.md

```diff
@@ -45,6 +45,7 @@ To quickly get started, see the [template interface](https://github.com/canonica
 |               | [`auth_proxy`](interfaces/auth_proxy/v0/README.md)                           |  ![Status: Draft](https://img.shields.io/badge/Status-Draft-orange)   | 
 |               | [`openfga`](interfaces/openfga/v0/README.md)                                 |  ![Status: Live](https://img.shields.io/badge/Status-Live-darkgreen)  | 
 |               | [`saml`](interfaces/saml/v0/README.md)                                       |  ![Status: Live](https://img.shields.io/badge/Status-Live-darkgreen)   |
+|               | [`smtp`](interfaces/smtp/v0/README.md)                                       |  ![Status: Live](https://img.shields.io/badge/Status-Live-darkgreen)   |
```

Comment:
are you sure you want to go straight to live and not put it in draft first? That means you can't change the interface anymore

Reply:
I'm sure. The spec defining it has been closed and the charm implementing it is already in `stable`